common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -41,6 +41,12 @@ class WordCloudFields(object):
         scope=Scope.settings,
         default="Word cloud"
     )
+    instructions = String(
+        display_name=_("Instructions"),
+        help=_("Add instructions for this word cloud"),
+        scope=Scope.settings,
+        default="List your five favorite foods."
+    )
     num_inputs = Integer(
         display_name=_("Inputs"),
         help=_("Number of text boxes available for students to input words/sentences."),
```

Comment:
@clrux Any chance we can sneak in a terminology fix in this PR? "students" -> "learners". Thanks!